osbs/utils.py

```diff
@@ -341,6 +344,9 @@ def make_name_from_git(repo, branch, limit=53, separator='-'):
     while name.endswith('-'):
```

Comment:
well it doesn't hurt to not have there multiple dashes "in the end" (before hash)